content/glance.pl.md

```diff
@@ -0,0 +1,130 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:"
+
+title: "ATAG w skrócie"
+nav_title: "ATAG w skrócie"
+lang: pl   # Change "en" to the translated language shortcode
+last_updated: 2024-03-05   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+translators:
+  - name: "Stefan Wajda"
+
+
+github:
+  repository: w3c/wai-intro-atag
+  path: content/glance.pl.md   # Add the language shortcode to the middle of the filename, for example: content/glance.fr.md
+  
+  
+permalink: /standards-guidelines/atag/glance/pl   
+ref: /standards-guidelines/atag/glance/   # Do not change this
+
+layout: default
+feedbackmail: wai@w3.org
+
+# In the footer below:
+# Do not change the dates
+# Translate the other words, including "Date:" and "Editors:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: |
+  <p><strong>Data:</strong> Uakualniono 18 maja 2016.</p>
+  <p><strong>Redaktorzy:</strong> Jeanne Spellman i <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Opracowane z udziałem Grupy Roboczej ds. dostępności narzędzi autorskich (<a href="https://www.w3.org/WAI/AU/">AUWG</a>) oraz Grupy Roboczej ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>).</p>
+
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Podsumowanie" class="full" %}
+{:/}
+
+Ta strona zawiera sparafrazowane podsumowanie Wytycznych dla dostępności narzędzi autorskich (ATAG) 2.0. Normatywna specyfikacja techniczna znajduje się na stronie
+[www.w3.org/TR/ATAG20](https://www.w3.org/TR/ATAG20/).
+
+**Aby dowiedzieć się więcej o ATAG 2.0, zapoznaj się z [[Omówieniem ATAG]](/standards-guidelines/atag/pl
```

Comment:
The line break before the closing parenthesis needs to be removed to fix the rendering of the link.

![Capture d’écran 2024-03-11 à 10 23 42](https://github.com/w3c/wai-intro-atag/assets/7482063/bbd95dba-e8d4-445f-abcd-3dc553c50f03)

Reply:
fixed

Reply:
@zwiastunsw Are you sure? I still see a closing parenthesis line 43, that should be at the end of line 42.

Reply:
@zwiastunsw Oh I have just spotted a remaining issue: please keep`/standards-guidelines/atag/` (rather than `/standards-guidelines/atag/pl` for it to work. The website will automatically link to the Polish version when published.

Reply:
@remibetin :  I was just about to ask this :smile:

Reply:
All good, thanks!